rollup.config.js

```diff
@@ -1,16 +1,17 @@
-import commonjs from 'rollup-plugin-commonjs';
+const commonjs = require('rollup-plugin-commonjs');
```

Comment:
Why?  rollup handles esm configs fine.  I think that's their preferred format.

Reply:
I think that's just a cut & paste artifact.